packages/contracts/contracts/MarketRegistry.sol

```diff
@@ -124,35 +131,38 @@ contract MarketRegistry is
     /**
      * @notice Creates a new market.
      * @param _initialOwner Address who will initially own the market.
-     * @param _paymentCycleDuration Length of time in seconds before a bid's next payment is required to be made.
+     * @param _paymentCycleValue Length of time in seconds before a bid's next payment is required to be made.
      * @param _paymentDefaultDuration Length of time in seconds before a loan is considered in default for non-payment.
      * @param _bidExpirationTime Length of time in seconds before pending bids expire.
      * @param _requireLenderAttestation Boolean that indicates if lenders require attestation to join market.
      * @param _requireBorrowerAttestation Boolean that indicates if borrowers require attestation to join market.
      * @param _paymentType The payment type for loans in the market.
      * @param _uri URI string to get metadata details about the market.
+     * @param _paymentCycleType The payment cycle type for loans in the market - Seconds or Monthly
      * @return marketId_ The market ID of the newly created market.
      */
     function createMarket(
         address _initialOwner,
-        uint32 _paymentCycleDuration,
+        uint32 _paymentCycleValue,
         uint32 _paymentDefaultDuration,
         uint32 _bidExpirationTime,
         uint16 _feePercent,
         bool _requireLenderAttestation,
         bool _requireBorrowerAttestation,
         PaymentType _paymentType,
+        PaymentCycleType _paymentCycleType,
```

Comment:
it might be better if we add this above the `_paymentCycleDuration` value so they are next to each other for end users